notebooks/official/tabular_workflows/wide_and_deep_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1144 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ! gcloud ai models delete $CUSTOM_JOB_MODEL --quiet
</pre><p>Use the Vertex SDK (i.e. model.delete(...)) instead of gcloud.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/984/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
Done.